FreeRTOS/FreeRTOS
# Add SMP schedule equal priority on target test

Add SMP schedule equal priority on target test

Description
-----------
Adding the on target test to verify that
> The scheduler shall schedule tasks of equal priority in a round robin fashion.

Test Steps
-----------
Test result
```
/FreeRTOS/FreeRTOS/Test/Target/tests/smp/schedule_equal_priority/schedule_equal_priority.c:183:Test_ScheduleEqualPriority:PASS

-----------------------
1 Tests 0 Failures 0 Ignored
```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
